runtime/sys.go

```diff
@@ -217,8 +217,8 @@ func readLinesFromFile(filepath string) []string {
 func isContainer() bool {
 	lines := readLinesFromFile(_cgroupPath)
 	for _, line := range lines {
-		if strings.HasPrefix(line, _dockerPath) ||
-			strings.HasPrefix(line, _kubepodsPath) {
+		if strings.Contains(line, _dockerPath) ||
```

Comment:
_cgroupPath like this:
14:name=systemd:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
13:rdma:/
12:pids:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
11:hugetlb:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
10:net_prio:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
9:perf_event:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
8:net_cls:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
7:freezer:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
6:devices:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
5:memory:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
4:blkio:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
3:cpuacct:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
2:cpu:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa
1:cpuset:/docker/edbb9148ff9af98f07c178359554e9de35a044337db4ac88f8c4a68fd955dafa

So, shoulde change it to _cgroupPath.Contains("_dockerPath")